### DISCUSSION TITLE
Feature Request: Expand `confirm-close-surface` to include a list of processes that don't need confirmation (matching Terminal.app)

### DISCUSSION THREAD

**Discussion by kevinastone** [ghostty-org/ghostty#3274] [Feature Requests, Ideas] [answered]
I like the feature in MacOS Terminal.app that allows you to provide a list of processes that can be running and quit without confirmation.  It lets me exit quickly out of ssh/tmux/etc but protects me from killing a running process I might not want to terminate.

<img width="440" alt="Screenshot 2024-12-26 at 9 40 34 PM" src="https://github.com/user-attachments/assets/44304508-c95b-47e9-90a6-0e816879147a" />

**Comment by mitchellh** [comment]
Thanks, I'm quite certain there is a duplicate of this already, but will keep this open until we find it.

**Reply by timvisher** [reply, marked as answer]
https://github.com/ghostty-org/ghostty/discussions/2677 mayhaps? :)

**Comment by net** [comment]
iTerm also supports a configurable list:

<img width="579" alt="Screenshot 2024-12-31 at 17 38 14" src="https://github.com/user-attachments/assets/feb47bb5-3c42-47fc-afe1-7be6fce0ef2e" />
<p>

At the very least, hard-coding a tmux exemption would go along way for any users using tmux.
